vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/authentication/validate_authentication_test.go

```diff
@@ -32,7 +32,7 @@ func TestFailValidateAuthenticationSpec(t *testing.T) {
 		},
 		"invalid webhook ref": {
 			spec: configv1.AuthenticationSpec{
-				WebhookTokenAuthenticators: []configv1.WebhookTokenAuthenticator{
+				WebhookTokenAuthenticators: []configv1.DeprecatedWebhookTokenAuthenticator{
```

Comment:
this field was deprecated in https://github.com/openshift/api/pull/666 not sure if these tests are still valid.
/cc @stlaz